universal-nft/docs/operations.md

```diff
@@ -0,0 +1,110 @@
+# Universal-NFT Operations
+
+This document describes how `mint` and `transfer` operations work within the Universal-NFT extension, and how indexers and tools are expected to interpret them.
+
+---
+
+## 1. Mint Operation
+
+### 1.1 Purpose
+
+Creates a new NFT on the Bitcoin blockchain and assigns its initial owner.
+
+### 1.2 Payload Example
+
+{
+  "p":   "nft",
+  "op":  "mint",
+  "tick":"example",
+  "meta":"abc",
+  "sig": "123",
+  "amt": "1"
+}
+
+### 1.3 Transaction Structure (Conceptual)
+
+A valid mint transaction typically includes:
+
+- One or more standard outputs (P2WPKH, P2TR, etc.)
+- One OP_RETURN output containing the JSON payload
+
+Indexers reconstruct state by:
+
+1. Scanning all transaction outputs  
+2. Detecting an OP_RETURN containing a valid Universal-NFT mint payload  
+3. Assigning ownership to the **first standard output** of the transaction  
```

Comment:
In universal it's the output after the OP_RETURN which is fitting with this only if OP_RETURN is output 0. 
Not first OP_RETURN are assigning to output 0?